docker-compose.yml

```diff
@@ -13,13 +13,14 @@ services:
     web:
         build: .
         container_name: same_lane_web
-        command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+        entrypoint: ["sh", "-c", "sleep infinity"]
```

Comment:
This should not be merged as this was Bill's testing code and would break the docker setup for everyone